### DISCUSSION TITLE
Best Surface to buy

### DISCUSSION THREAD

**Discussion by Kobaratz** [linux-surface/linux-surface#1276] [Q&A]
Hi,

I'm interested in buying a cheap surface to run linux on, but I'm not sure which one's have the best support for my needs as of 2023. I've checked the feature matrix but i'm still unsure, mainly because I don't need much from the tablet so would prefer not to get something overly expensive!

Ideally:
- Good pen and touch support for note taking
- Can view media on it and isn't too slow (mostly just anime/shows)
- I'm not too bothered about screen size as I mostly need it for note taking
- I may look to get Waydroid setup on it so if anyone has experience setting that up on linux surfaces, happy to hear about it

Don't need much more than that..is the Surface Go 3 best? Is there much performance difference between the pentium and i3 model?

**Comment by qzed** [comment]
I guess the Surface Go line (Go 3 or now maybe even Go 4) isn't bad if you don't need too much power. The only thing I'd personally look out for there is eMMC vs proper SSD (i.e., go for the SSD if you can afford it, eMMC tends to be somewhat slow). I can't speak to what kind of power/CPU you need for waydroid, so maybe someone else can help out with that.

**Comment by shafai** [comment]
I wish support for Pro X was better to be able to recommend it. Battery life is amazing on ARM, but a lot of things don't work.
